From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Batman <batman@Batman.local>
Date: Sun, 16 Aug 2026 00:43:39 +0200
Subject: [PATCH] browse patch 001: branding (product name, company, bundle id)

---
 chrome/app/theme/chromium/BRANDING | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index f8363d5b294fe..6c592c6161f87 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
-COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
-MAC_CREATOR_CODE=Cr24
+COMPANY_FULLNAME=LAN-SOLO UG
+COMPANY_SHORTNAME=LAN-SOLO
+PRODUCT_FULLNAME=browse
+PRODUCT_SHORTNAME=browse
+PRODUCT_INSTALLER_FULLNAME=browse Installer
+PRODUCT_INSTALLER_SHORTNAME=browse Installer
+COPYRIGHT=Copyright @LASTCHANGE_YEAR@ LAN-SOLO UG and The Chromium Authors. All rights reserved.
+MAC_BUNDLE_ID=com.lan-solo.browse
+MAC_CREATOR_CODE=Brws
 MAC_TEAM_ID=
